Cargo.toml

```diff
@@ -68,3 +68,5 @@ rustc-demangle = { opt-level = 3 }
 [patch.crates-io]
 parquet2 = { version = "0.13", optional = true, git = "https://github.com/datafuse-extras/parquet2", branch = "parquet2-0.13-patch2" }
 chrono = { git = "https://github.com/datafuse-extras/chrono", rev = "279f590" }
+# https://github.com/calder/rust-goldenfile/pull/7
+goldenfile = { git = "https://github.com/psiace/rust-goldenfile", rev = "16c5783" }
```

Comment:
Can we use a crate, not a GitHub repo?

Reply:
This is just a patch approach, and once the upstream merge is done, we can say goodbye to the github repo.

Or, if necessary, I can maintain a new crate.

Reply:
It doesn't worth implementing a new crate for such change. If upstream reject or no-responding to our PR, we can implement [`Differ`](https://docs.rs/goldenfile/latest/goldenfile/differs/type.Differ.html) instead.

Reply:
@BohuTANG Do those explanations make sense to you?  Please take a look again.